config/issuetracker.go

```diff
@@ -23,6 +24,7 @@ var ValidIssueTrackerAuthTypes = map[IssueTracker][]AuthType{
 	IssueTrackerPivotal: {AuthTypeNone, AuthTypeAPIToken},
 	IssueTrackerRedmine: {AuthTypeNone, AuthTypeAPIToken},
 	IssueTrackerJira:    {AuthTypeNone, AuthTypeOffline},
+	IssueTrackerAzure:   {AuthTypeNone, AuthTypeAPIToken},
```

Comment:
Do azure boards support none auth type?

Reply:
Yes, you can view public boards/items without auth token.
Confirmation screenshot:
![image](https://user-images.githubusercontent.com/22592783/124609948-c35c9b00-de78-11eb-9d79-fd2670d9626d.png)